app/main/views.py

```diff
@@ -1,7 +1,21 @@
 from flask import render_template
 from . import main
+from ..models import Resource
+import json
+from flask import jsonify
 
-
-@main.route('/')
+@main.route('/', methods=['GET'])
 def index():
     return render_template('main/index.html')
+
+@main.route('/get-resource', methods=['GET'])
+def getResource():
```

Comment:
nit: naming convention - `get_resource` instead of `getResource` (the former is more pythonic)